test/error-message-lister.js

```diff
@@ -0,0 +1,242 @@
+        code:              TYPE.emptyTypeActionArgument,
+        stepName:          'Step',
+        relatedSourceCode: 'code'
+    },
+    {
+        code:     TYPE.unexpectedDialog,
+        stepName: 'Step',
+        dialog:   'test-dialog',
+        message:  'message'
+    },
+    {
+        code:     TYPE.expectedDialogDoesntAppear,
+        stepName: 'Step',
+        dialog:   'test-dialog'
+    },
+    {
+        code:              TYPE.incorrectSelectActionArguments,
+        stepName:          'Step',
+        relatedSourceCode: 'code'
+    },
+    {
+        code:              TYPE.incorrectWaitActionMillisecondsArgument,
+        stepName:          'Step',
+        relatedSourceCode: 'code'
+    },
+    {
+        code:              TYPE.incorrectWaitForActionEventArgument,
+        stepName:          'Step',
+        relatedSourceCode: 'code'
+    },
+    {
+        code:              TYPE.incorrectWaitForActionTimeoutArgument,
+        stepName:          'Step',
+        relatedSourceCode: 'code'
+    },
+    {
+        code:              TYPE.waitForActionTimeoutExceeded,
+        stepName:          'Step',
+        relatedSourceCode: 'code'
+    },
+    {
+        code:              TYPE.emptyIFrameArgument,
+        stepName:          'Step',
+        relatedSourceCode: 'code'
+    },
+    {
+        code:              TYPE.iframeArgumentIsNotIFrame,
+        stepName:          'Step',
+        relatedSourceCode: 'code'
+    },
+    {
+        code:              TYPE.multipleIFrameArgument,
+        stepName:          'Step',
+        relatedSourceCode: 'code'
+    },
+    {
+        code:              TYPE.incorrectIFrameArgument,
+        stepName:          'Step',
+        relatedSourceCode: 'code'
+    },
+    {
+        code:              TYPE.uploadCanNotFindFileToUpload,
+        stepName:          'Step',
+        relatedSourceCode: 'code',
+        filePaths:         ['path1', 'path2']
+    },
+    {
+        code:              TYPE.uploadElementIsNotFileInput,
+        stepName:          'Step',
+        relatedSourceCode: 'code',
+        filePath:          'path'
+    },
+    {
+        code:              TYPE.uploadInvalidFilePathArgument,
+        stepName:          'Step',
+        relatedSourceCode: 'code'
+    }
+];
+
+var testMock = {
+    name:    'fixtureTest',
+    fixture: {
+        name: 'fixture',
+        path: './fixture.js'
+    }
+};
+
+var browserConnectionMocks = [
+    { userAgent: 'Chrome' }
+];
+
+// Task mock
+var TaskMock = function () {
+    EventEmitter.call(this);
+
+    this.tests              = [testMock];
+    this.browserConnections = browserConnectionMocks;
+};
+
+util.inherits(TaskMock, EventEmitter);
+
+var testRunMock = {
+    test:              testMock,
+    unstable:          false,
+    browserConnection: browserConnectionMocks[0],
+    errs:              errs
+};
+
+/* eslint-disable no-console */
+console.log('Enter the reporter type - spec | list | xunit | json | minimal:');
```

Comment:
We shouldn't list all reporters here, since the list may change, so it should be something like:
`Enter reporter (e.g. spec, xunit):`
\cc @VasilyStrelyaev